.idea/codeStyles/Project.xml

```diff
@@ -1,11 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <JetCodeStyleSettings>
-      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```

Comment:
don't change this

Reply:
I have noticed that there might have been some automatic changes made by Android Studio itself. I will investigate this further to identify and verify any modifications that have occurred.